app-i18n/transifex-client/transifex-client-0.13.11.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A command line interface for Transifex"
+HOMEPAGE="https://pypi.org/project/transifex-client/ https://www.transifex.net/"
+SRC_URI="mirror://pypi/t/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
+RDEPEND="dev-python/GitPython[${PYTHON_USEDEP}]
+	<dev-python/python-slugify-5.0.0[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	<dev-python/six-2.0.0[${PYTHON_USEDEP}]
+	dev-python/urllib3[${PYTHON_USEDEP}]"
+
+distutils_enable_tests setup.py
```

Comment:
```
Ran 103 tests in 0.215s

FAILED (failures=1)
Test failed: <unittest.runner.TextTestResult run=103 errors=0 failures=1>
error: Test failed: <unittest.runner.TextTestResult run=103 errors=0 failures=1>
 * ERROR: app-i18n/transifex-client-0.13.11::testworld failed (test phase):
 *   Tests fail with python3.6
```

Looks good but could you check whether this works for you? You can disable it this way:
https://dev.gentoo.org/~mgorny/python-guide/test.html#skipping-problematic-tests

Reply:
Thanks for the hint. I excluded the failing test.